### DISCUSSION TITLE
keyball44:トラックボールを被覆銅線で繋ぎたい

### DISCUSSION THREAD

**Discussion by DD-masa** [Yowkees/keyball#206] [Q&A]
**目的**
トラックボールの動作に必要な回路を被覆銅線で繋ぎたいので、どことどこを繋げたらよいか教えていただきたいです。

**経緯**
以前までは問題なく使用できていましたが、
トラックボールの動きを認識しずらくなってしまったので
はんだをやりなおしたところ、今度は反応すらしなくなってしましました。
10時間ほど関係していそうな箇所をいじっていたのですが、直らずでした。
今回は被覆銅線を購入したので、被覆銅線をつなげることで解決したいです。
トラックボールの動作に必要な箇所で、どことどこをつなげればよいか教えていただきたいです。

![pcb](https://user-images.githubusercontent.com/129586819/229278250-67d75ab1-f873-4c76-9680-8846060d4913.jpg)
![ballpcb](https://user-images.githubusercontent.com/129586819/229278209-0cd37c84-5deb-43cb-8032-86f33292a71d.jpg)

**Comment by Yowkees** [comment]
![trackball_fix1](https://user-images.githubusercontent.com/84209772/229944383-68efa5ad-d9b4-42e4-8acb-994d19c112ea.jpg)
状況了解しました。
まず、赤丸部がショートしてしまっているようです。この状態ですとトラックボールは動きませんので、半田こてでこの部分を切り離してみてください。
お手数ですがご対応お願いいたします。

**Reply by DD-masa** [reply]
@Yowkees 
ありがとうございます。
こちら対応しました。

**Reply by DD-masa** [reply]
@Yowkees
こちら対応しましたが、依然トラックボール動作しませんので、どこを繋げるべきか教えていただきたいです。

**Comment by Yowkees** [comment]
電子回路はどこを繋ぐかと同時に、つながってはいけない箇所を切り離す必要があります。
不具合の多くは後者が原因です。

7Pコンスルーと近くのジャンパ4か所の半田付けは写真を見る限り大丈夫そうですので、まずはProMicro裏の半田付けで隣とくっついている個所がないかご確認ください。ここはスルーホールがジグザグに配置されており、隣との距離が近いのでショートしやすいです。
テスターを使うとより確実です。ご確認お願いいたします。

また、下記にて回路図と基板図を公開しています。原因究明に役立てていただければ幸いです。
https://github.com/Yowkees/keyball/tree/main/keyball44/design_data
